recipes/default.rb

```diff
@@ -3,6 +3,8 @@
 # Recipe:: default
 #
 
-%w( repo install configure service ).each do |r|
+Chef::Recipe, Chef::Resource].each { |l| l.send :include, ::Extensions }
```

Comment:
yes is syntax error sorry :-).
Library is loaded first, so we don't need to load them inside other recipes called after (plugins).
I use this approach because i based to other cookbook elastic/cookbook-elasticsearch and extract part of their code to include in your cookbook this feature.